src/Ressources/views/form_admin_fields.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends 'SonataSonataHelpersBundle:Form:form_admin_fields.html.twig' %}
```

Comment:
Are you sure about the bundle name? Seems weird

Reply:
namespace issue is not clear for now